docs/development/tests.rst

```diff
@@ -17,14 +17,26 @@ environments by running::
 
     tox
 
-In order to run all test including the search tests, include `"'--including-search'"`
-argument::
+By default, tox won't run tests from search,
+in order to run all test including the search tests,
+you need to override tox's posargs.
+If you don't have any additional arguments to pass,
+you can also set the ``TOX_POSARGS`` environment variable to an empty string::
 
-    tox "'--including-search'"
+    TOX_POSARGS='' tox
+
+.. note::
+
+   If you need to override tox's posargs, but you still don't want to run the search tests,
+   you need to include ``-m 'not search'`` to your command::
+
+       tox -- -m 'not search' -x
```

Comment:
I liked the idea of using the `--including-search` that was before this PR, which by default added `-m 'not search'` automatically by inspecting the `-m` argument first.

If I understand correctly, now when running `tox -- -k mytest` we will be forced to add `-m 'not search'` as well.

Reply:
Can we keep this behavior?

Reply:
I wasn't able to make it work with .com. And if `tox -- -k mytest` doesn't match a test from search you don't have to add `-m 'not search'`